CHANGELOG.md

```diff
@@ -129,7 +130,7 @@
 
 ### Internals
 * Sync protocol version bumped to 4. ([#5382](https://github.com/realm/realm-core/pull/5382))
-* Internal sync metadata tables were abstracted into a new schema management framework and their schema versions are now tracked in the `sync_internal_schemas` table. ([#5455](https://github.com/realm/realm-core/pull/5455)) 
+* Internal sync metadata tables were abstracted into a new schema management framework and their schema versions are now tracked in the `sync_internal_schemas` table. ([#5455](https://github.com/realm/realm-core/pull/5455))
```

Comment:
I think your editor auto formatted this file. We normally encourage big formatting changes to be addressed separately, but since I already verified all these whitespace changes I don't mind them as a drive by fix.

Reply:
Yes, my editor is aggressively removing white spaces. I leave it as it is.